### PR TITLE
Muhammad_togrladi

### PR DESCRIPTION
**Pull request by MuhammadMangitov**

### CODE REVIEW

**Review by MuhammadMangitov — COMMENTED** [review]
mmm

File: Lesson01/Lesson01/Program.cs [hunk 1]
```diff
@@ -11,7 +11,9 @@ static void Main(string[] args)
         // Complete the following method
         public static int Add(int a, int b)
         {
-            return 0;
+
```
Comment (Mirazyzz) [inline Lesson01/Lesson01/Program.cs:14]: Bo'shliq kerak emas,

File: Lesson01/Lesson01/Program.cs [hunk 2]
```diff
@@ -11,7 +11,9 @@ static void Main(string[] args)
         // Complete the following method
         public static int Add(int a, int b)
         {
-            return 0;
+           
+            return a + b;
+            //return a + b;
```
Comment (Mirazyzz) [inline Lesson01/Lesson01/Program.cs:16]: Komentariya ham keraksiz

**Comment by Mirazyzz** [comment]
Pull Requestni nomini to'g'irla

File: Lesson01/Lesson01/Program.cs (hunk 2 above)
Comment (MuhammadMangitov) [reply Lesson01/Lesson01/Program.cs]: Togrlandi

**Review by Mirazyzz — APPROVED** [review]
Approved!

**Review by JavlonAlimov — APPROVED** [review]

**Pull request closed without merging**